src/test/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelperTest.java

```diff
@@ -165,62 +168,75 @@ public void testSaveReportForStudy() throws Exception {
+                .withReportData(TEST_REPORT.getData()).build();
 
-        bridgeHelper.saveReportForStudy(TEST_STUDY_ID, TEST_REPORT_ID, TEST_REPORT);
+        bridgeHelper.saveReportForStudy(report);
         verify(mockCall).execute();
     }
-
+    
     @Test
-    public void testSessionHelper() throws Exception {
-        // 3 test cases:
-        // 1. first request initializes session
-        // 2. second request re-uses same session
-        // 3. third request gets 401'ed, refreshes session
-        //
-        // This necessitates 4 calls to our test server call (we'll use save report):
-        // 1. Initial call succeeds.
-        // 2. Second call also succeeds.
-        // 3. Third call throws 401.
-        // 4. Fourth call succeeds, to complete our call pattern.
-
-        // Mock ForWorkersApi - third call throws
-        Call<Message> mockReportCall1 = mock(Call.class);
-        Call<Message> mockReportCall2 = mock(Call.class);
-        Call<Message> mockReportCall3a = mock(Call.class);
-        Call<Message> mockReportCall3b = mock(Call.class);
-        when(mockReportCall3a.execute()).thenThrow(NotAuthenticatedException.class);
-
-        ForWorkersApi mockWorkersApi = mock(ForWorkersApi.class);
-        when(mockWorkersApi.saveReportForStudy(TEST_STUDY_ID, TEST_REPORT_ID, TEST_REPORT)).thenReturn(mockReportCall1,
-                mockReportCall2, mockReportCall3a, mockReportCall3b);
-
-        // Mock AuthenticationApi
-        Call<UserSessionInfo> mockSignInCall = mock(Call.class);
-        AuthenticationApi mockAuthApi = mock(AuthenticationApi.class);
-        when(mockAuthApi.signIn(TEST_SIGN_IN)).thenReturn(mockSignInCall);
-
-        // Mock Client Manager
+    public void testGetParticipantsForStudy() throws Exception {
+        ForWorkersApi mockWorkerClient = mock(ForWorkersApi.class);
+        
         ClientManager mockClientManager = mock(ClientManager.class);
-        when(mockClientManager.getClient(AuthenticationApi.class)).thenReturn(mockAuthApi);
-        when(mockClientManager.getClient(ForWorkersApi.class)).thenReturn(mockWorkersApi);
+        when(mockClientManager.getClient(ForWorkersApi.class)).thenReturn(mockWorkerClient);
 
-        // Set up Bridge Helper
+        AccountSummary summary1 = new AccountSummary().id(USER_ID_1).email(USER_EMAIL_1);
+        AccountSummary summary2 = new AccountSummary().id(USER_ID_2).email(USER_EMAIL_2);
+        
+        Call<AccountSummaryList> mockCall1 = createResponseForOffset(0L, 100L, summary1, summary2);
+        when(mockWorkerClient.getParticipantsInStudy(TEST_STUDY_ID, null, 100, null, TEST_START_DATETIME,
+                TEST_END_DATETIME)).thenReturn(mockCall1);
+        
+        Call<AccountSummaryList> mockCall2 = createResponseForOffset(100L, null);
+        when(mockWorkerClient.getParticipantsInStudy(TEST_STUDY_ID, 100, 100, null, TEST_START_DATETIME,
+                TEST_END_DATETIME)).thenReturn(mockCall2);
```

Comment:
Is it worth having at least 2 pages of data to make sure that pagination is working?